release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2645,3 +2645,175 @@ Issued: 2020-06-15
+
+* The *View more* link on the *Installed Operators* page now links to the
+correct page.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1824255[*BZ#1824255*])
+
+* With this release, the *Status* column on the operand view in the web console
+is updated to show the latest status that is available in the *Details* and
+*YAML* views.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1831808[*BZ#1831808*])
+
+* With this release, the `eventSources` API Group is updated to the latest
+supported API Group, `sources.knative.dev`. This update allows sources generated
+by the new API Group to be recognized in the *Topology* view of the web console.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1836807[*BZ#1836807*])
+
+* Previously, environment variables for Knative service could not be specified
+from the *Add* view of the *Developer* perspective in the web console. As a
+result, applications that required environment variables might not work as
+expected. With this release, users can add environment variables from the *Add*
+view.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1839115[*BZ#1839115*])
+
+* Previously, the web console did not display user details when the user name
+contained special characters such as `#`. The web console now displays user
+details regardless of special characters in the user name.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1840812[*BZ#1840812*])
+
+* After upgrading Octavia from OpenStack 13 to 16, the UDP listener is supported
+and the strategy to enforce DNS resolution over TCP protocol is removed. This
+change requires adding the new listener to the existent DNS service that
+specifies the UDP protocol. The old Amphora image for the existent DNS load
+balancer does not support the new listener and causes the listener creation to
+fail. With this release, the DNS service that requires UDP is re-created,
+causing the load balancer to be re-created with the new Amphora version.
+Re-creating the service and load balancer causes some down time for DNS
+resolution. When this process is complete, the load balancer for the DNS service
+is created with all the required listeners.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1841029[*BZ#1841029*])
+
+* Previously, the web console did not create the role binding required for the
+OpenShift Monitoring Prometheus Operator to collect operator metrics when the
+`operatorframework.io/cluster-monitoring=true` annotation was set to `true`. The
+issue is resolved in this release.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1841149[*BZ#1841149*])
+
+* Previously, autoscaler required provider IDs across Node and Machine objects
+to match exactly, and if the Machine configuration specified a resource group
+name with mixed casing, the ID was not an exact match. In this situation,
+autoscaler did not recognize that a Machine had a Node and terminated the
+Machine after 15 minutes. With this release, autoscaler ignores the case of
```

Comment:
I agree with capitalizing `Node` and `Machine` in the first sentence since they're being specifically referred to as objects. I read their following usages as a more generic reference, which would not require capitalization. As you've probably found, these capitalization rules are weird and confusing (:stuck_out_tongue_closed_eyes:), but I would probably lower-case the remaining `node` and `machine` references after the first sentence.